src/engines/quickjs.js

```diff
@@ -38,8 +40,15 @@ class QuickJSInstaller extends Installer {
   }
 
   getDownloadURL(version) {
+    const baseVersion = '6.0.0';
```

Comment:
you should update `resolveVersion` to work based on the platform instead of hard coding a version.

Reply:
fixed.